Update projects (#537)
elevenlabs/elevenlabs-docs: product/projects/overview.mdx

> Click "Add a new project” and you will be presented with a popup. Here, you can choose to create a new empty project, import an already existing EPUB, PDF, TXT or HTML file, which will then automatically be converted into a project, or import text directly from a website using the URL to have the page be converted into a project. You can then use our [Audio Native](/product/audio-native/overview) feature to easily and effortlessly embed any narration project onto your website.

Reviewer: nit: For step-by-step workflows I would suggest using Mintlfiy [Steps](https://mintlify.com/docs/content/components/steps#steps)

Author: I will add Steps to this section on a new branch to keep things clean!

If you want to convert the entire chapter in one go, you can click the convert button in the upper right corner. This will open a page where you can choose to convert either your entire project or individual chapters. You can also download the entire project or individual chapters. Even after converting the whole chapter, you can still go back and regenerate sections of the book that you are not happy with before downloading the entire thing. However, if you make any changes, you will need to press convert once again for the changes to be reflected in the whole book, so you can download the entire chapter.

<Frame>
  <img height="200" src="/product/projects/images/projects_convert.webp" />
</Frame>

After the conversion of either a whole project or individual chapters has finished, you will be able to see these conversions by clicking “Versions” next to either the project or the individual chapters. You can then download the different versions.

Once your Project is converted, you have several download options available. 

<Frame>
  <img height="200" src="/product/projects/images/projects_zip_dropdown.webp" />
</Frame>

## Auto-Regenerate

> When you have auto-regenerate enabled for your project, we will automatically check the output for any mispronunciations or unwanted audio artefacts.  If we detect any, we will automatically regenerate the audio up to twice, at no extra cost.  

Reviewer: ```suggestion
When the Auto-Regenerate feature is enabled for your project, we will automatically check the output for any mispronunciations or unwanted audio artefacts.  If we detect any, we will automatically regenerate the audio up to twice, at no extra cost.  
```
